packages/babel-plugin-redux-saga/src/index.js

```diff
@@ -1,31 +1,36 @@
 var SourceMapConsumer = require('source-map').SourceMapConsumer
 var pathFS = require('path')
-var importHelper = require('@babel/helper-module-imports')
 
-var fnId = 'reduxSagaSource'
-var tempVarId = 'res'
-var locationSymbolPath = 'redux-saga'
-var locationSymbolExportName = 'SAGA_LOCATION'
-var hintedLocationSymbolExportName = locationSymbolExportName
+var globalSymbolNames = {location: '@@redux-saga/LOCATION'}
+// var globalSymbolNames = require('redux-saga').globalSymbolNames
+
+var wrapperFunctionName = 'reduxSagaSource'
+var tempVarName = 'res'
+var symbolName = globalSymbolNames.location
+var plainPropertyName = globalSymbolNames.location
 
 module.exports = function(babel) {
   var { types: t } = babel
   var sourceMap = null
-  var symbolNameBinding = null
 
-  function getAssignementWithSymbolProperty(name, symbolName) {
-    return t.memberExpression(t.identifier(name), t.identifier(symbolName), true)
+  function getAssignementWithSymbolProperty(name) {
+    return t.memberExpression(
```

Comment:
I have lately really enjoyed using `@babel/template`, you can access it from the `babel` object here too, without need to import it yourself):
```j
var { types: t, template } = babel
```

We can only use `template` (no `template.ast`, `template.expression`, etc) to keep babel@6 compatibility.

IMHO it improves the readability quite a lot.